broker/lib/cdn.py

```diff
@@ -12,3 +27,188 @@ def is_cdn_instance(service_instance) -> bool:
 
 def is_cdn_dedicated_waf_instance(service_instance) -> bool:
     return isinstance(service_instance, CDNDedicatedWAFServiceInstance)
+
+
+def parse_alarm_notification_email(instance, params):
+    if not is_cdn_dedicated_waf_instance(instance):
+        return None
+
+    return params.get("alarm_notification_email")
+
+
+def parse_cache_policy(params, cache_policy_manager: CachePolicyManager) -> str:
+    cache_policy = params.get("cache_policy", None)
+    if not cache_policy:
+        return None
+    if cache_policy not in config.ALLOWED_AWS_MANAGED_CACHE_POLICIES:
```

Comment:
this is a duplicate of the test we run when loading the cache policies

Reply:
Hmm, yeah, it is, but I do like the idea of throwing a meaningful error to users if they specify a non-supported value rather than just silently doing nothing. Thoughts?